utils/get_images.py

```diff
@@ -57,9 +58,6 @@ def find_target_tweets(trend_one, tweets_with_images):
 
     return trend_one[0], str(id_num), image_url
```

Comment:
eventually we will want to do this for all trends -- and further filter this afterwards. Using metadata we receive from tweets_with_images

Reply:
Agreed